python/lsst/obs/base/cameraMapper.py

```diff
@@ -1213,7 +1213,7 @@ def _standardizeExposure(self, mapping, item, dataId, filter=True,
             exposure = exposureFromImage(item, dataId, mapper=self, logger=self.log,
                                          setVisitInfo=setVisitInfo, setFilter=filter)
         except Exception as e:
-            self.log.error("Could not turn item=%r into an exposure: %s" % (repr(item), e))
+            self.log.error("Could not turn item=%r into an exposure: %s", item, e)
```

Comment:
That `repr` is probably there because we don't have good `str` for many objects, but that's something we should just fix when it comes up.

Reply:
Yes. I was a bit confused why we were calling repr twice: once with `repr` and once with `%r`...

Reply:
Oh, ha! I didn't even notice that. Good catch.